R/kwic.R

```diff
@@ -64,7 +64,7 @@ kwic.character <- function(x, pattern, window = 5, valuetype = c("glob", "regex"
 #' @export 
 kwic.corpus <- function(x, pattern, window = 5, valuetype = c("glob", "regex", "fixed"), 
                         case_insensitive = TRUE, join = FALSE, ...) {
-    kwic(texts(x), pattern, window, valuetype, case_insensitive, join, ...)
+    kwic(tokens(x, ...), pattern, window, valuetype, case_insensitive, join)
```

Comment:
This bypasses the checks for deprecated arguments in `kwic.character()`, and means we call the same thing twice, from two different functions. Better with the original.